bbb-common-web/src/main/java/org/bigbluebutton/api/Util.java

```diff
@@ -113,6 +116,23 @@ public static File getPresFileDownloadMarker(File presBaseDir, String presId, St
 		return null;
 	}
 
+	public static void deleteAlreadyMarkedDownloadableFiles(File presFileDir, String presId) {
+		String regexString = "\\.(" + String.join("|",
+				SupportedFileTypes.getSupportedFileTypes()) + ")\\.downloadable";
+		File[] filesWithDownloadMark = presFileDir.listFiles(new FilenameFilter() {
+			@Override
+			public boolean accept(File dir, String name) {
+				return name.matches(presId + regexString);
+			}
+		});
+
+		for (File file: filesWithDownloadMark) {

```

Comment:
Here too -- looking at the naming it seems we're deleting the presentation file, not the "mark". Let's adjust to make it clearer we're deleting the mark. A line comment may also be welcome.

Reply:
Good suggestion, what do you think about `downlaodableMarks` only? I think it seems clear that we are only selecting the marks to be deleted and not the presentation files itself